src/Components/Spotify/TokenExtractor.js

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
It's not a component, so should be called `tokenExtractor.js` (and maybe be elsewhere, but idk)

Reply:
maybe we make a folder with all the remaining scripts or API related things?

Reply:
The website that recommended this structure had a `Utils` folder, but it could also be somewhere with the spotify files